app/Controllers/ScBookingController.cs

```diff
@@ -143,6 +143,14 @@ public async Task<IActionResult> BookingType(ScBookingTypeViewModel model)
                     );
                 }
 
+                if (model.EstimatedTrialLength > 99)
+                {
+                    ModelState.AddModelError(
+                        "EstimatedTrialLength",
+                        "Estimated trial length must be between 1 and 99 days."
+                    );
```

Comment:
@LL-scj-gov is 99 days a realistic limit for online bookings, or should we make it even shorter? And perhaps we could instruct people to contact the registry for longer trials?

Reply:
The managers have decided to go with 40 as the top number.  Anything beyond that gets into complicated breaks due to court calendar items.

Reply:
Thanks Lorne!